yurim/week1/완주하지못한선수.js

```
@@ -0,0 +1,33 @@
+    participant.sort();
+    completion.sort();
+
+    for(let i = 0; i < participant_length; i++) {
+        if(participant[i] !== completion[i]){
+            return participant[i];
+        }
+    }
+}
+
+// 정답률 91.7ㄴ
+function solution(participant, completion) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'solution'. Consider to delete it or rename it.

'solution' is defined here:


(lint/suspicious/noRedeclare)